### PR TITLE
Feat/rulespg

### PR DESCRIPTION
**Pull request by aurobindoGupta**
image

### CODE REVIEW

**Review by ckahub — COMMENTED** [review]
LGTM

File: components/rulespg/styles.css [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+}
+.navBar {
+  background: var(--background-color);
+}
+.box-container {
+  background: var(--background-color);
+  display: block;
+  position: static;
+  width: 50%;
+  box-shadow: 3px 3px 7px 1px rgba(0, 0, 0, 0.3),
```
Comment (ckahub) [inline components/rulespg/styles.css:44]: use rem everywhere

**Review by ckahub — COMMENTED** [review]
LGTM

**Review by devdeadviz — APPROVED** [review]
Everything looks good to me!

File: components/quePg/styles.css [hunk 2]
```diff
@@ -64,10 +64,10 @@
   transform: scale(0.98);
 }
 .rightAns {
-  background: green;
+  background: var(--stdGreen-color);
```
Comment (smilegupta) [inline components/quePg/styles.css:67]: could have used a better name for variable

File: components/quePg/styles.css [hunk 3]
```diff
@@ -64,10 +64,10 @@
...
   color: var(--background-color);
 }
 .wrongAns {
-  background: rgb(226, 30, 30);
+  background:var(--stdRed-color);
```
Comment (smilegupta) [inline components/quePg/styles.css:71]: same as above

File: components/rulespg/index.html [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <meta name="author" content="Aurobindo Gupta" />
+  <meta name="description" content="Personal Quizz Website" />
+  <link rel="icon" type="image/x-icon" href="Utils/images/iconpanda16.ico" />
+
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" />
+  <link rel="stylesheet" href="https://panda-ui-neog.netlify.app/rootStyles.css" />
```
Comment (smilegupta) [inline components/rulespg/index.html:13]: could have included these both in your styles.css

File: components/rulespg/styles.css (hunk 1 above)
Comment (smilegupta) [inline components/rulespg/styles.css:44]: post every css class can leave a line and can have used rem

**Review by smilegupta — COMMENTED** [review]
Added some sanity comments

**Pull request merged**